packages/field-views-loader/index.js

```diff
@@ -62,60 +86,66 @@ module.exports = function() {
 
   let allPaths = new Set();
 
-  const stringifiedObject = serialize(
-    Object.entries(adminMeta.lists).reduce((obj, [listPath, { views }]) => {
+  let pageComponents = findPageComponents(adminMeta.pages);
+
+  let allViews = Object.entries(adminMeta.lists).reduce(
+    (obj, [listPath, { views }]) => {
       obj[listPath] = views;
       return obj;
-    }, {}),
-    allPaths
+    },
+    { __pages__: pageComponents }
```

Comment:
Could the pages be another export? Or maybe the same object but with `pages` and `lists` keys?

Reply:
I'm planning to do a more significant refactor of the loader, including getting rid of the lists export altogether and exporting views per field type instead which will reduce the bundle size quite a bit for projects with lots of lists and fields.

I'd rather do that refactoring with another PR, and keep this to just the addition of custom pages into the current export.

Reply:
> including getting rid of the lists export altogether and exporting views per field type instead which will reduce the bundle size quite a bit for projects with lots of lists and fields.

I just thought about this more and realized that it has a problem, the views of a field type can change for each for each usage of a field type. While this isn't true for most field types, it's true for one, the Content field type since the blocks defined for a specific usage of the field type determine the views.